openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreQuickFilter.tsx

```diff
@@ -88,12 +88,9 @@ const ExploreQuickFilter: FC<ExploreQuickFilterProps> = ({
       getAdvancedFieldOptions(query, index, advancedField)
         .then((res) => {
           const suggestOptions =
-            // TODO: Fix type issues below
-            // eslint-disable-next-line @typescript-eslint/no-explicit-any
-            (res.data as any).suggest['metadata-suggest'][0].options ?? [];
+            res.data.suggest['metadata-suggest'][0].options ?? [];
```

Comment:
Can we create a util for this to get the options from API response and use it here and in other places?

Reply:
I will do this in separate PR